src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioErrorReportingService.cs

```diff
@@ -31,43 +30,72 @@ internal partial class VisualStudioErrorReportingService : IErrorReportingServic
             _listener = listener;
         }
 
-        public void ShowErrorInfo(string message, params ErrorReportingUI[] items)
+        public void ShowErrorInfoInActiveView(string message, params ErrorReportingUI[] items)
```

Comment:
The problem with 'active view' is that it means that some messages show up on top of the error-list (for example).  This is because we're reporting an error for some issue, and hte user just happens to have the error list selected.  I've run into this, and it's not good.

I don't think this is ever a good idea.  We should always just be using the top-most yellow bar.  

As there's no connection between what we're doing in the background and the 'active view', trying to tie them together just makes the experience weird.